src/RA_Defs.h

```diff
@@ -11,26 +14,67 @@
 #include <queue>
 #include <deque>
 #include <map>
+//#include <memory>
+//#include <thread>
+//#include <mutex>
 
 #ifndef RA_EXPORTS
 
 //	Version Information is integrated into tags
 
 #else
+// if you really want to use null instead of nullptr like .net
+#ifndef null
+#define null nullptr
+#endif // !null
```

Comment:
We're not trying to be .NET. c++ has `NULL` and `nullptr`. We use those.

Reply:
ok

Reply:
it's done